ethcore/src/types/executed.rs

```diff
@@ -20,13 +20,14 @@ use util::numbers::*;
 use util::Bytes;
 use trace::{Trace, VMTrace};
 use types::log_entry::LogEntry;
+use types::state_diff::StateDiff;
 use ipc::binary::BinaryConvertError;
 use std::fmt;
 use std::mem;
 use std::collections::VecDeque;
 
 /// Transaction execution receipt.
-#[derive(Debug, PartialEq, Clone, Binary)]
+#[derive(Debug, PartialEq, Clone)]
```

Comment:
`Executed` is in `types` directory, so I assume that it should implement `Binary`. It's probably not an issue right now, cause no tests are failing, but it may interfere with Nikolay's pull requests and might be difficult to debug, because compiler plugins errors are hardly readable.

imo, we should keep derive from `Binary` and also implement it for `StateDiff`